2-structured-data/app.yaml

```diff
@@ -11,5 +11,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-runtime: nodejs
-env: flex
+runtime: nodejs8
+
+handlers:
+- url: /.*
+  script: auto
```

Comment:
newline

Reply:
Done: https://github.com/GoogleCloudPlatform/nodejs-getting-started/blob/app-engine-std/2-structured-data/app.yaml

Reply:
Funny - I still see the No Newline at end indicator here.

Reply:
Yeah :( for the some reason the preview system decides to go with 6993a7dcc0 (3 days ago) instead of the latest commit.